doc/js_tutorials/js_setup/js_setup/js_setup.markdown

```diff
@@ -127,6 +127,13 @@ Building OpenCV.js from Source
     python ./platforms/js/build_js.py build_js --build_test
     @endcode
 
+-#  [optional] To build support for Aruco markers from the OpenCV contrib modules append `--cmake_option="-DOPENCV_EXTRA_MODULES_PATH=/path/to/opencv_contrib/modules/"`
```

Comment:
No need to highlight only Aruco. Just "To add contrib modules, add a flag"

Reply:
Wow you are quick  ^^

Reply:
Done